ga/developer/javaee7/Dockerfile

```diff
@@ -14,6 +14,14 @@
 
 FROM websphere-liberty:webProfile7
 
+ARG CUSTOM_REPO=false
+ARG CUSTOM_REPO_HOSTNAME=lars
+
 COPY server.xml /config/
-RUN installUtility install --acceptLicense defaultServer \
+
+RUN if [ "$CUSTOM_REPO" = true ] ; then echo "useDefaultRepository=false\nLARS.url=http://$CUSTOM_REPO_HOSTNAME:9080/ma/v1\nLARS.user=admin\nLARS.password=adminPwd" > /opt/ibm/wlp/etc/repositories.properties; fi \
```

Comment:
I don't like the fact that this codifies lots of things about our particular usage of LARS in to the Dockerfile. One option would  be to have a bunch of ARGS for the URL, user, password, ... but I think it would be cleaner just to have a single ARG that was the contents of repositories.properties (REPOSITORIES_PROPERTIES?). You can also ditch the CUSTOM_REPO flag and just do something like:
````
    RUN if [ ! -z $REPOSITORIES_PROPERTIES]; then echo $REPOSITORIES_PROPERTIES > ...
````